101-sonarqube-docker-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,452 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "adminUsername": {
+      "type": "string",
+      "minLength": 1,
```

Comment:
is this the actual min length? For some reason I have 3 in my head, but I don't know for sure. 1 seems rather low to me